public/cli/lib/rendering/recordFrames.spec.ts

```diff
@@ -0,0 +1,28 @@
+import test from 'ava';
+import tempy from 'tempy';
+import fs from 'fs';
+import { record } from './recordFrames';
+// TODO: Add a puppeteer mock model
+test('recordFrames: render 5 mock frames', async (t) => {
+  let htmlContent = createMockHtml();
+  const numberOfFrames = 5;
+  await tempy.directory.task(async (outputLocation) => {
+    let files = fs.readdirSync(outputLocation);
```

Comment:
It would make the test clearer if you made this `const` and renamed `files` to make it clearer what files are being tested. A few lines down you would make a new `const` with a clearer name of what is being tested there.

Reply:
done.